CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
+### Changed
+
+- Prefer solutions that pass the `wrap_column` the least.
```

Comment:
I think this would be clearer if it clarified that this is more of a 'tie-breaker' between solutions that violate the line length limit the same number of times.